### ISSUE TITLE
Error loading command module 'cosmosdb'

### ISSUE THREAD

**Issue by berndverst** [Azure/azure-cli#3556]
### Description
Whenever I attempt to switch subscriptions via `az account set --subscription` the CLI returns the error `Error loading command module 'cosmosdb'`.

Details from the debug logs:
```
Traceback (most recent call last):
  File "/Users/bverst/Library/Python/2.7/lib/python/site-packages/azure/cli/core/commands/__init__.py", line 287, in get_command_table
    import_module('azure.cli.command_modules.' + mod).load_commands()
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/azure/cli/command_modules/cosmosdb/__init__.py", line 14, in load_commands
    import azure.cli.command_modules.cosmosdb.commands  # pylint: disable=redefined-outer-name
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/azure/cli/command_modules/cosmosdb/commands.py", line 24, in <module>
    APPLICATION.register(APPLICATION.COMMAND_PARSER_PARSING, deprecate)
AttributeError: 'Application' object has no attribute 'COMMAND_PARSER_PARSING'
```

---

### Environment summary

**Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
Answer here: `pip install azure-cli`

**CLI Version:** What version of the CLI and modules are installed? (Use `az --version`)  
Answer here: `2.0.6`

**OS Version:** What OS and version are you using?  
Answer here: Mac OS 10.12.5

**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, Bash on Windows)  
Answer here: ZSH

**Comment by derekbekoe** [comment]
Was this installed into a clean virtual environment? Looks like there's a mismatch of dependencies.

What's the full output of `az --version`?
Also, what's the output of `pip freeze | grep azure-cli-`?

**Comment by berndverst** [comment]
No it's installed globally.

```
az --version
azure-cli (2.0.6)

acr (2.0.2)
acs (2.0.4)
appservice (0.1.4)
batch (2.0.2)
cdn (0.0.2)
cloud (2.0.2)
cognitiveservices (0.1.2)
command-modules-nspkg (2.0.0)
component (2.0.2)
configure (2.0.4)
container (0.1.4)
core (2.0.4)
cosmosdb (0.1.6)
dla (0.0.6)
dls (0.0.6)
documentdb (0.1.4)
feedback (2.0.2)
find (0.2.1)
interactive (0.3.2)
iot (0.1.4)
keyvault (2.0.2)
lab (0.0.3)
monitor (0.0.3)
network (2.0.4)
nspkg (3.0.0)
profile (2.0.4)
rdbms (0.0.1+dev)
redis (0.2.1)
resource (2.0.4)
role (2.0.3)
sf (1.0.1)
shell (0.2.0)
sql (2.0.2)
storage (2.0.4)
vm (2.0.4)

Python (Darwin) 2.7.13 (default, Dec 18 2016, 07:03:39)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)]

Python location '/usr/local/opt/python/bin/python2.7'
```

```
pip freeze | grep azure-cli-
azure-cli-acr==2.0.2
azure-cli-acs==2.0.4
azure-cli-appservice==0.1.4
azure-cli-batch==2.0.2
azure-cli-cdn==0.0.2
azure-cli-cloud==2.0.2
azure-cli-cognitiveservices==0.1.2
azure-cli-command-modules-nspkg==2.0.0
azure-cli-component==2.0.2
azure-cli-configure==2.0.4
azure-cli-container==0.1.4
azure-cli-core==2.0.4
azure-cli-cosmosdb==0.1.6
azure-cli-dla==0.0.6
azure-cli-dls==0.0.6
azure-cli-documentdb==0.1.4
azure-cli-feedback==2.0.2
azure-cli-find==0.2.1
azure-cli-interactive==0.3.2
azure-cli-iot==0.1.4
azure-cli-keyvault==2.0.2
azure-cli-lab==0.0.3
azure-cli-monitor==0.0.3
azure-cli-network==2.0.4
azure-cli-nspkg==3.0.0
azure-cli-profile==2.0.4
azure-cli-rdbms==0.0.1+dev
azure-cli-redis==0.2.1
azure-cli-resource==2.0.4
azure-cli-role==2.0.3
azure-cli-sf==1.0.1
azure-cli-shell==0.2.0
azure-cli-sql==2.0.2
azure-cli-storage==2.0.4
azure-cli-vm==2.0.4
```

**Comment by derekbekoe** [comment]
It appears there are multiple instances of the CLI on the machine:
- Your initial comment had `/Users/bverst/Library/Python/2.7/lib/python/site-packages/`
- Yet `az --version` shows `/usr/local/opt/python/bin/python2.7`

What about `type -a az`? What directory does that give?

**Comment by berndverst** [comment]
I now remember messing with the CLI's own virtualenv when I had to load a module which had not been released yet at that time. Looks like I messed up a bunch of things.

I just tried to wipe the CLI -- had to run the following multiple times
`pip freeze | grep azure-cli- | cut -d'=' -f1 | xargs pip uninstall -y`

And then I installed via `curl -L https://aka.ms/InstallAzureCli | bash` -- I let it wipe out my previous installation.

It's all good now. Sorry for the false alarm.
